src/worker/users.js

```diff
@@ -218,6 +231,21 @@ class Users {
         await network.save();
         return network;
     }
+
+    async expireUserTokens(createdExpire, accessedExpire) {
+        return this.db.dbUsers('user_tokens')
```

Comment:
I'm not sure what the aim of the `.where(fn)` is here, are you expecting to enclose the raw SQL `WHERE` clause within `()`?

If not, the query builder can be split up such as:
~~~javascript
       let q = this.db.dbUsers('user_tokens').delete();
       if (createdExpire) {
           q.where('created_at', '<=', createdExpire);
           ...
       } else {
           q.where('accessed_at', '<=', accessedExpire);
       }
       return q;
~~~